src/commands/math/commands.js

```diff
@@ -314,31 +314,7 @@ LatexCmds['^'] = P(SupSub, function(_, super_) {
   };
 });
 
-var SummationNotation = P(MathCommand, function(_, super_) {
-  _.init = function(ch, html) {
-    var htmlTemplate =
-      '<span class="mq-large-operator mq-non-leaf">'
-    +   '<span class="mq-to"><span>&1</span></span>'
-    +   '<big>'+html+'</big>'
-    +   '<span class="mq-from"><span>&0</span></span>'
-    + '</span>'
-    ;
-    Symbol.prototype.init.call(this, ch, htmlTemplate);
-  };
-  _.createLeftOf = function(cursor) {
-    super_.createLeftOf.apply(this, arguments);
-    if (cursor.options.sumStartsWithNEquals) {
-      Letter('n').createLeftOf(cursor);
-      Equality().createLeftOf(cursor);
-    }
-  };
-  _.latex = function() {
-    function simplify(latex) {
-      return latex.length === 1 ? latex : '{' + (latex || ' ') + '}';
-    }
-    return this.ctrlSeq + '_' + simplify(this.ends[L].latex()) +
-      '^' + simplify(this.ends[R].latex());
-  };
```

Comment:
there's a bug in this parsing, it produces 2 child blocks for both sums and limits, you can see this bug by doing:
```js
mathField.latex('\\lim').keystroke('Left').typedText('x').latex()
```
You should get `\lim_x`, but you get `\lim_{ }` instead. You'll find that any `\lim` rendered from LaTeX, moving around with the left and right arrow keys is messed up because of the invisible second block (what in summation notation is the upper limit block).

Did you guys do this factoring this way because you it was too difficult to figure out how to write your own parser? We've gotten a lot of feedback that it's quite inscrutable, is that your experience as well?